school_attendance/models/school_attendance.py

```diff
@@ -381,14 +377,9 @@ def _compute_absent(self):
                         count_fail += 1
                 rec.total_absent = count_fail
 
-    @api.constrains('date')
-    def validate_date(self):
-        if self.date > date.today():
-            raise ValidationError(_("Date should be less than or equal to\
-            current date!"))
-
     date = fields.Date("Date", help="Current Date",
-                       default=lambda *a: time.strftime('%Y-%m-%d'))
+                       default=lambda *a: time.strftime(
+                                            DEFAULT_SERVER_DATE_FORMAT))
```

Comment:
fields.Date.today

Reply:
It was updated in following commit [31334b9](https://github.com/JayVora-SerpentCS/OdooEduERP/pull/184/commits/31334b949711ee516c3968636c608149d586336c).